src/utils/tracer.rs

```diff
@@ -161,15 +168,12 @@ where
     ///
     /// Use this to delay edge addition in the underlying graph. Recover all
     /// traced additions at the end of tracing using [`into_edge_additions`](Self::into_edge_additions).
-    pub fn add_edge(&mut self, constraint: EdgeWeight, from_node: TracedNode, to_state: NodeId) {
+    pub fn add_action(&mut self, action: Action, node: TracedNode) {
```

Comment:
I _feel_ the parameters should be inversed here; so we write `.add_action(here, with this value)` like we do in portgraph.

But I can't find cases for either style in this codebase, so no need to change it.